_episodes/07-photo.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Photometry"
-teaching: 55
+teaching: 40
 exercises: 0
```

Comment:
Zero time here is unexpected.

Reply:
This is correct. This episode has no exercises. My next task is to look through the pilot notes to see where some could be added.